### ISSUE TITLE
Re-design shuffle / repeat icons

### ISSUE THREAD

**Issue by Taiko2k** [Taiko2k/TauonMusicBox#1037]
The current repeat and shuffle buttons while visually unique may be un-intuitive for new users. A new design may be better. Feel free to give your feedback and or submit a design proposal. 

Considerations:

 - Both repeat and shuffle buttons
 - Icons must be in svg format. Monochromatic with alpha background.
 - A wide aspect ratio may be preferred as to match closer to the existing hitboxes.
 - Mode variants:
   - Repeat Track
   - Repeat Album
   - Shuffle Tracks
   - Shuffle Albums

**Comment by Solarunit** [comment]
I like the current unique buttons, I don't have a problem with them, I mostly use Repeat track.

**Comment by skylineone044** [comment]
Design proposal:
![tauon_repeat_shuffle_v1](https://github.com/Taiko2k/TauonMusicBox/assets/37515608/312a2f17-0cbb-4e12-adcf-922d8ef5353c)

This includes all 4 variants, and the emblems on the left can be excluded form each one if need be.

**Comment by Taiko2k** [comment]
Wow that's really cool, if you can put each of those as an SVG I can try it out. Keep the icon on the left in seperate svgs, and I might only use the album varients, not the track ones.

**Comment by skylineone044** [comment]
Sure, here they are:
[tauon_shuffle_repeat_icon_components.zip](https://github.com/Taiko2k/TauonMusicBox/files/13245529/tauon_shuffle_repeat_icon_components.zip)

**Comment by Taiko2k** [comment]
Ok Ive committed this change now. Thanks for contribution! 

Ive taken some creative liberty to what I think might be more visually readable. 

![Screenshot from 2023-11-03 14-00-04](https://github.com/Taiko2k/TauonMusicBox/assets/17271572/36ef571a-5873-4c9a-b200-a1c62088fb5e)

![Screenshot from 2023-11-03 14-02-56](https://github.com/Taiko2k/TauonMusicBox/assets/17271572/9cbed7ad-e77d-4508-93e3-755b0d557249)

I'm open to any feedback.

**Comment by Solarunit** [comment]
How they look in disabled state?
I think it looks okay, but imo shuffle icon on top is too wide, I like the one at the bottom.
I think the arrow shape-part of the "left" (repeat) and "right" (shuffle) icons should be the same width..

**Comment by Taiko2k** [comment]
When disabled they look the same as the top set.

One benefit of it being wide is it maintains the same general shape between modes. And it fills the wide hitbox area.

**Comment by skylineone044** [comment]
The 1 and the album are a bit above the arrow on the repeat icon, I think they should be centered:
![tauon_repeat_a](https://github.com/Taiko2k/TauonMusicBox/assets/37515608/f041c9bb-0cf5-4b14-9b70-59722bb5fcbe)
![tauon_repeat](https://github.com/Taiko2k/TauonMusicBox/assets/37515608/ebe08980-a35f-4faf-a003-ea75b04224b2)

**Comment by Solarunit** [comment]
> The 1 and the album are a bit above the arrow on the repeat icon, I think they should be centered:

I agree

> One benefit of it being wide is it maintains the same general shape between modes. And it fills the wide hitbox area.

I mean making it the same with as shorter one but with another shuffle state indicator, similar to the 1 or album circle


____

Other idea: make  state indicator, like 1 and album symmetrical, e.g. by moving album to the right of the shuffle arror

**Comment by StrangeSeasOfThought** [comment]
The shuffle icon is so wiiiiiide. It shouldn't be (that much) wider than the repeat icon imo, or any other icon. It's like 1/3 of the volume bar, looks really strange.

**Comment by Solarunit** [comment]
I agree, it should be the same width as the repeat icon in inactive state

**Comment by C0rn3j** [comment]
Related issue #1021
